plugins/modules/zos_copy.py

```diff
@@ -0,0 +1,1513 @@
+      - If C(src) is a directory, destination must be a partitioned data set.
+      - If C(dest) is a nonexistent path, it will be created.
+      - If C(dest) is a nonexistent data set, it will be allocated.
+      - If C(src) and C(dest) are files and if the parent directory of C(dest)
+        does not exist, then the task will fail.
+      - When the C(dest) is VSAM(KSDS) or VSAM(ESDS), then source can be ESDS,
+        KSDS or RRDS.
+      - When the C(dest) is VSAM(RRDS), then the source must be RRDS.
+      - When C(dest) is VSAM(LDS), then source must be LDS.
+    type: str
+    required: true
+  content:
+    description:
+      - When used instead of C(src), sets the contents of a file or data set
+        directly to the specified value.
+      - Works only when C(dest) is a USS file, sequential data set, or a
+        partitioned data set member.
+      - This is for simple values, for anything complex or with formatting, use
+        the M(template) module.
+      - If C(dest) is a directory, then content will be copied to
+        C(/path/to/dest/inline_copy).
+    type: str
+    required: false
+  backup:
+    description:
+      - Specifies whether a backup of destination should be created before
+        copying data.
+      - When set to C(true), the module creates a backup file or data set.
+    type: bool
+    default: false
+    required: false
+  backup_file:
+    description:
+      - Specify the USS file name or data set name for the destination backup.
+      - If the destination (dest) is a USS file or path, the backup_file name must be a file
+        or path name, and the USS path or file must be an absolute path name.
+      - If the destination is an MVS data set, the backup_file name must be an MVS
+        data set name.
+      - If the backup_file is not provided, the default backup_file name will
+        be used. If the destination is a USS file or USS path, the name of the backup
+        file will be the destination file or path name appended with a
+        timestamp, e.g. C(/path/file_name.2020-04-23-08-32-29-bak.tar).
+      - If the destination is an MVS data set, it will be a data set with a random
+        name generated by calling the ZOAU API. The MVS backup data set
+        recovery can be done by renaming it.
+    required: false
+    type: str
+  force:
+    description:
+      - If set to C(true), the remote file or data set will be overwritten.
+      - If set to C(false), the file or data set will only be copied if the destination
+        does not exist.
+      - If set to C(false) and destination exists, the module exits with a note to
+        the user.
+    type: bool
+    default: true
+    required: false
+  mode:
+    description:
+      - The permission of the destination file or directory.
+      - If C(dest) is USS, this will act as Unix file mode, otherwise ignored.
+      - Refer to the M(copy) module for a detailed description of this
```

Comment:
@ddimatos  Since the `mode` parameter behaves essentially identical to the `copy` modules, would it make sense to pull the description from its doc? Would it cause any licensing issues?

Reply:
@asifmahmud @RadhaV2020 looks like you two already resolved this, would be no license issue, sorry for the delay, must have missed the email notification.